src/app/components/client/LocationAutocomplete.module.scss

```
@@ -0,0 +1,16 @@
+@import "../../tokens";
+
+.locationItem {
+  font: $text-body-sm;
+  padding: $spacing-sm calc($spacing-sm * 1.5);
+
+  strong {
+    letter-spacing: 0.01em;
+  }
+
+  span {
+    color: $color-grey-40;
+    font: $text-body-xs;
+    margin-left: 0.5em;
```

suggestion: This should be a token _(This may need a different token than what is suggested below)._
```suggestion
    margin-left: $spacing-sm
```

---

I used `em` intentionally so the spacing is in relation to the font size.

---

Added a comment about it: https://github.com/mozilla/blurts-server/pull/3217/commits/1b195d87775e45b003abe06b666fbd4b4a4f0aea.